.coafile

```diff
@@ -0,0 +1,18 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js
+
+[all.markdown]
+bears = MarkdownBear
+files = *.md
+
+[all.json]
+bears = JSONFormatBear
+files = *.json
+
+[all.js]
+bears = ESLintBear
+files = *.js, app/**.js, config/*.js, tests/**.js
```

Comment:
I'm not sure if ESLint will run without `.eslintrc`

Also no problem at all on the js code?

Reply:
add https://github.com/ember-cli/eslint-plugin-ember ?

Reply:
The default `eslint:recommended` preset doesn't include style rules, IMO maybe use external config like [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb) or similar.

Also, there's PrettierLintBear that lint style only, I think it should cover what `eslint:recommended`  is missing.

Reply:
Note that Prettier will most likely destroy all your current coding style, which might make huge changes.

Reply:
I'm okay with that, if it good for the project. Will have plenty of time in 2nd and third phase of GSoC to do that.

But it is for later improvement. As current ESLint And ESLint ember plugin is good enough 

This PR, I want to provide basic support for coafile.

Reply:
Ok, specify [`eslint_config`](https://github.com/coala/gci-leaders/blob/master/.coafile#L23) seems like it's necessary, the config is not read if it's not specified explicitly.